docker-compose.yml

```diff
@@ -140,9 +144,11 @@ services:
       - handshake
 
   dnslink-api:
-    build:
-      context: ./packages/dnslink-api
-      dockerfile: Dockerfile
+    # uncomment "build" and comment out "image" to build from sources
+    # build:
+    #   context: https://github.com/SkynetLabs/skynet-webportal.git#master
```

Comment:
is this context correct?

Reply:
surprisingly yes https://github.com/docker/compose/pull/8933